vscode-dotnet-runtime-library/src/EventStream/EventStreamEvents.ts

```diff
@@ -1754,3 +1754,8 @@ export class TestAcquireCalled extends IEvent
         return undefined;
     }
 }
+
+function getDisabledTelemetryOnChance(percentIntToSend: number): { [disableTelemetryId: string]: boolean }
+{
+    return { suppressTelemetry: Math.random() * 100 < percentIntToSend ? true : false };
```

Comment:
Took a peek at what Math.random() does in Javascript. So, this'll only be a value between 0 and 99 inclusive. Which is why it works for `< 1` as 1%. However, that predicate is a boolean itself. So, the `? true : false` is redundant.
```suggestion
    return { suppressTelemetry: Math.random() * 100 < percentIntToSend };
```

Reply:
Great point, ty.